2.2/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openshift/base-centos7
+
+# This image provides a Ruby 2.2 environment you can use to run your Ruby
+# applications.
+
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
```

Comment:
@danmcp @smarterclayton is there some redhat email alias we can use as the MAINTAINER for our images?

i'd consider openshift@redhat.com but if ultimately these are maintained by the SCL team that's probably not the right answer.